index.js

```diff
@@ -16,6 +16,139 @@ cookieButton.addEventListener('click', () => {
      cookie.classList.add('hidden');
 })
 
+//active navbar elements
+
+function activateMenuLink(allLinks, linkToActivate) { //funkcja dodajaca active do linkow  (pierwszy parametr lista wszystkich linkow, w drugim link ktory chce aktywowac)
+     for(let i = 0; i < allLinks.length; i++){ //petla dla calej listy linkow
+
+          let link = allLinks[i]; // element w tablicy all links
+
+          if (link.classList.contains('active')) { //  jesli jest active to usuwamy
+               link.classList.remove("active");
+          }
+     }
+
+     if (!linkToActivate.classList.contains('active')) { // jesli nie zawiera active to dodajemy
+          linkToActivate.classList.add("active");
+     }
+
+     let href = linkToActivate.getAttribute("href"); //pobieramy atrybut href (np #features) 
+
+     window.history.pushState(href, "Vitamina - "+href, href); // dodajemy go to historii przegladarki- zmienia sie url bez reloadu
```

Comment:
Czy to oczekiwane zachowanie? Mnie to denerwuje jak cos mi nadpisuje historie, klikam potem back na myszce, i zamiast wrocic na poprzednia strone, to latam po calej stronie jak glupi